settings.gradle

```diff
@@ -0,0 +1,2 @@
+rootProject.name = 'spring-webservice'
+
```

Comment:
The below file "src/images/tips.jpg" should not be stored in github repository. Please delete it.